pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -105,6 +105,11 @@ self: super: builtins.intersectAttrs super {
     configureFlags =  "--extra-include-dirs=${pkgs.rdkafka}/include/librdkafka";
   });
 
+  hw-kafka-client = overrideCabal super.hw-kafka-client (drv: {
+    preConfigure = "sed -i -e /extra-lib-dirs/d -e /include-dirs/d -e /librdkafka/d hw-kafka-client.cabal";
```

Comment:
Please add a comment at the top explaining why those lines need to be deleted. Also, it would be nice to report that issue upstream (and possibly add a comment referring to that Github ticket, too).